app/dts/bindings/zmk,animation.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2020, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: Global animation configuration
+
+compatible: "zmk,animation"
+
+properties:
+  drivers:
+    type: phandles
+    required: true
+    description: |
+      This array should contain all driver devices responsible for illuminating animated LEDs.
+      The devices must implement Zephyr's LED Strip Interface and expose a chain-lenght devicetree property.
```

Comment:
I'm confused why we required `chain-length` to be specified as a property of the LED strip device. The upstream LED strip driver bindings do not all expose this property (only the WS2818 seems to). Could we instead use a separate integer array of chain lengths within the root animation node to encode this? Something like this:
```
animation {
		compatible = "zmk,animation";
		drivers = <&led_ctrl1 &led_ctrl2>;
		chain-lengths = <128 128>;
};
```

Reply:
You're right, there aren't really any rules surrounding `led_strip` devices so that seems like a better option indeed.